### ISSUE TITLE
Why no hactoberfest participation ?

### ISSUE THREAD

**Issue by jainendra** [conwnet/github1s#362]
It would have been good to see this listed for hacktoberfest. It would have brought more engagement and participation

**Comment by jainendra** [comment]
@xcv58 Yeah, I am registered. I was asking why don't we add a topic `hacktoberfest` in this repo. This will then be eligible for hacktoberfest and people's contirbution will be counted in the hacktoberfest challenge hence bringing in more engagement

**Comment by conwnet** [comment]
> @xcv58 Yeah, I am registered. I was asking why don't we add a topic `hacktoberfest` in this repo. This will then be eligible for hacktoberfest and people's contirbution will be counted in the hacktoberfest challenge hence bringing in more engagement

Thank you so much for the information! I have already add the `hacktoberfest` tag in this repo.🚀
